codec-http/src/main/java/io/netty/handler/codec/http/HttpServerUpgradeHandler.java

```diff
@@ -353,8 +353,7 @@ private boolean upgrade(final ChannelHandlerContext ctx, final FullHttpRequest r
      * Creates the 101 Switching Protocols response message.
      */
     private static FullHttpResponse createUpgradeResponse(CharSequence upgradeProtocol) {
-        DefaultFullHttpResponse res = new DefaultFullHttpResponse(HTTP_1_1, SWITCHING_PROTOCOLS,
-                Unpooled.EMPTY_BUFFER, false);
+        DefaultFullHttpResponse res = new DefaultFullHttpResponse(HTTP_1_1, SWITCHING_PROTOCOLS, Unpooled.EMPTY_BUFFER);
```

Comment:
How about adding a constructor with `boolean` and set the default value to `true` for enabling Header verification? It'll give the user ability to turn off Header verification if needed.

Reply:
Sure, if it's okay to introduce a new API in the `4.1`, then I can add a new constructor for `HttpServerUpgradeHandler`.

Reply:
It's fine to add new API. Add 2 constructors actually. 1 will be default constructor (empty) which will set Boolean value to true and 1 constructor will allow to set Boolean value.

Reply:
I think one should be enough since the class already has a few constructors. They will enable the validation by default. The new one will allow a user to turn the validation off.

Reply:
Yeah, alright. My bad, I didn't saw other constructor.